pkg/master/ports/ports.go

```diff
@@ -41,4 +42,7 @@ const (
 	// ProxyHealthzPort is the default port for the proxy healthz server.
 	// May be overridden by a flag at startup.
 	ProxyHealthzPort = 10256
+	// KubeControllerManagerPort is the default port for the controller manager status server.
+	// May be overridden by a flag at startup.
+	KubeControllerManagerPort = 10257
```

Comment:
We're claiming a port.  @liggitt does that require an api review or something?

Reply:
I don't actually know